.rubocop.yml

```diff
@@ -0,0 +1,9 @@
+AllCops:
+  Include:
+    - '**/Berksfile'
+    - '**/Cheffile'
+  Exclude:
+    - 'metadata.rb'
+
+Metrics/AbcSize:
+  Max: 9001
```

Comment:
https://www.youtube.com/watch?v=SiMHTK15Pik

ABC. There was a lot of logic in that one function.

http://c2.com/cgi/wiki?AbcMetric